packages/web-app-files/src/composables/resourcesViewDefaults/useResourcesViewDefaults.ts

```diff
@@ -11,7 +11,7 @@ import {
   useStore
 } from '@ownclouders/web-pkg'
 import { determineSortFields as determineResourceTableSortFields } from '@ownclouders/web-pkg'
-import { determineSortFields as determineResourceTilesSortFields } from '../../helpers/ui/resourceTiles'
+import { determineSortFields as determineResourceTilesSortFields } from '@ownclouders/web-pkg/src/helpers/ui/resourceTiles'
```

Comment:
When importing from `web-pkg` from any other package than `web-pkg` itself, please use `@ownclouders/web-pkg`.

```suggestion
import { determineSortFields as determineResourceTilesSortFields } from '@ownclouders/web-pkg'
```

For this to work you need to export `determineSortFields` in `web-pkg` via the index file in `packages/web-pkg/src/helpers/ui/index.ts`. Since you will get a naming conflict then (because there's 2 exported methods named `determineSortFields`), I'd suggesting renaming the actual functions to `determineResourceTableSortFields` and `determineResourceTilesSortFields`.